tests/SampleAppZeroBase/host/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../SampleAppZeroBase.edl)
```

Comment:
Since this is not a sample, zerobase_host, zerobase_enc would be more appropriate names for the host, enclave, test etc.

Reply:
done.